packages/govtnz-ds-website/src/components/SkipLink.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import './skip-link.scss';
+
+type SkipLinkProps = {
+  href: string;
+};
+
+const SkipLink = ({ href }: SkipLinkProps): JSX.Element => {
+  const handleClick = e => {
```

Comment:
We should wrap this in a `useCallback`

Reply:
@holloway interesting, I can see that `useCallback` is using memoization in the background. But I'm not sure how this will benefit this handler. Does it mean that if I put the handler inside `useCallback` it won't re-recreate the handler function again and just use the `handleClick` in memory?

Reply:
@janzenz that's right. it's a cache. whether that cache is beneficial is based on how often this component is called. see https://kentcdodds.com/blog/usememo-and-usecallback/